app/templates/book.hbs

```diff
@@ -2,10 +2,11 @@
 
 <BookDetails @book={{this.model}} />
 
-<LinkTo @route="edit-book" @model={{@model}}>Edit</LinkTo>
-<a href="" {{on "click" this.deleteBook}}>Delete</a>
-
-<LinkTo @route="new-review" @model={{@model}}>Add a review</LinkTo>
+{{#if this.currentUser.isAdmin}}
+  <LinkTo @route="edit-book" @model={{@model}}>Edit</LinkTo>
+  <a href="" {{on "click" this.deleteBook}}>Delete</a>
```

Comment:
nitpick, this should be a `<button>` element instead of a link. They can be styled as buttons.